google-beta/resource_pubsub_topic.go

```diff
@@ -15,17 +31,27 @@ func resourcePubsubTopic() *schema.Resource {
 		Delete: resourcePubsubTopicDelete,
 
 		Importer: &schema.ResourceImporter{
-			State: resourcePubsubTopicStateImporter,
+			State: resourcePubsubTopicImport,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(240 * time.Second),
+			Delete: schema.DefaultTimeout(240 * time.Second),
 		},
 
 		Schema: map[string]*schema.Schema{
 			"name": {
 				Type:             schema.TypeString,
 				Required:         true,
 				ForceNew:         true,
-				DiffSuppressFunc: linkDiffSuppress,
+				DiffSuppressFunc: comparePubsubTopicBasename,
```

Comment:
afaik `linkDiffSuppress` is a predecessor to `compareSelfLinkOrResourceName`, so let's use that instead.